code/modules/shuttle/custom_shuttle.dm

```diff
@@ -161,9 +192,25 @@
 	//This should really be accelleration, but its a 2d spessman game so who cares
 	if(calculated_mass == 0)
 		return FALSE
-	calculated_acceleration = (calculated_dforce*1000) / (calculated_mass*100)
+	calculated_acceleration = calculated_dforce / calculated_mass * CUSTOM_SHUTTLE_ACCELERATION_SCALE
 	return TRUE
 
 /obj/machinery/computer/shuttle_flight/custom_shuttle/connect_to_shuttle(obj/docking_port/mobile/port, obj/docking_port/stationary/dock, idnum, override=FALSE)
 	if(port && (shuttleId == initial(shuttleId) || override))
 		linkShuttle(port.id)
+
+/obj/machinery/computer/shuttle_flight/custom_shuttle/attackby(obj/item/I, mob/living/user, params)
+	if(istype(I, /obj/item/shuttle_creator) && !designator)
+		if(!user.transferItemToLoc(I,src))
+			return
+		designator = I
+		playsound(src, 'sound/machines/terminal_insert_disc.ogg', 50, 0)
```

Comment:
Add in . = ..() to this proc if the item isn't a shuttle designator, otherwise it will not be able to be attacked